projects/igniteui-angular/src/lib/core/styles/components/grid-summary/_grid-summary-theme.scss

```diff
@@ -165,6 +175,14 @@
         min-width: rem(30px);
         margin-inline-end: rem(3px);
 
+        @if $variant == 'indigo-design' {
+            margin-inline-end: initial;
+
+            /* stylelint-disable scss/at-extend-no-missing-placeholder */
+            @extend .ig-typography__caption;
```

Comment:
Why don't you use the type-style mixin instead to avoid having to disable linting rules?